script.js

```diff
@@ -0,0 +1,37 @@
+// Initializing an array for storing the quiz questions
+let quizQuestions = [];
+
```

Comment:
For good practice I would use the [eventListener DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event) when you have to interact with the DOM.

Reply:
excuse me Nicolai, do you mean it was not necessary to use this event listener and I have to remove it?